src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -65,6 +69,43 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
       return meUserInterestsLoader?.().then(({ body }) => body)
     },
   },
+  interestsConnection: {
+    type: UserInterestConnection,
+    args: pageable({}),
+    resolve: async (
+      { id, partnerId },
+      args,
+      { partnerCollectorProfileUserInterestsLoader }
+    ) => {
+      if (!partnerCollectorProfileUserInterestsLoader || !partnerId) {
+        return null
+      }
+
+      const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+      const {
+        body,
+        headers,
+      } = await partnerCollectorProfileUserInterestsLoader(
```

Comment:
**suggestion:** adding defense here would make more sense to prevent breaks. Maybe try catch or alternative

Reply:
yes, but we never do it in MP, all the paginated loaders we query I only kept the style but can move in this direction if you want to.

Reply:
That's true, but IMO risky, nevermind then!